src/main/java/lottogame/domain/lotto/Lottos.java

```diff
@@ -18,18 +20,24 @@ public Lottos(List<Lotto> lottos) {
         this.lottos = lottos;
     }
 
-    public Map<Rank, Integer> calculateWinningStatistics(Lotto winningLotto, LottoNumber bonusNumber) {
+    public Map<Rank, Integer> calculateWinningStatistics(WinningLotto winningLotto) {
         return lottos.stream()
-                .map(lotto -> Rank.from(lotto.countMatchingNumbers(winningLotto), lotto.isMatchingBonus(bonusNumber)))
+                .map(lotto -> Rank.from(lotto.countMatchingNumbers(winningLotto.getWinningLotto()), lotto.isMatchingBonus(winningLotto.getBonusNumber())))
```

Comment:
Nit: `WinningLotto`를 기준으로 당첨 결과를 비교하도록 변경해보고 차이점을 비교해보시는 것도 좋을 것 같습니다.
```suggestion
                .map(lotto -> Rank.from(winningLotto.countMatchingNumbers(lotto), winningLotto.isMatchingBonus(lotto)))
```